app/models/manageiq/providers/embedded_ansible/automation_manager/event_catcher/runner.rb

```diff
@@ -1,3 +1,12 @@
 class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::EventCatcher::Runner < ManageIQ::Providers::BaseManager::EventCatcher::Runner
   include ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::EventCatcher::Runner
+
+  def start_event_monitor
+    tid = super
+    return tid unless tid.nil?
+
+    # Get a new copy of the ems record in case the embedded ansible role changed servers
+    @ems.reload
```

Comment:
Spoke with @Fryguy and @agrare, we think this shouldn't be an issue for the refresher.